package.json

```diff
@@ -185,6 +185,15 @@
           "webpack"
         ]
       },
+      {
+        "packages": [
+          "@fluentui/babel-make-styles"
+        ],
+        "dependencies": [
+          "tslib",
```

Comment:
If this merges before the TS 4 upgrade you should add `nohoist` settings as well for `"@fluentui/babel-make-styles/tslib", "@fluentui/babel-make-styles/typescript"`

Reply:
Thanks for noticing this, added in a59bab0 👍